res/values/strings.xml

```diff
@@ -413,6 +413,9 @@
     <string name="start_app">Start…</string>
     <!-- this is a warning that is shown when one tries to send something to a chat that is not yet accepted. -->
     <string name="accept_request_first">Please accept the chat request first.</string>
+    <!-- placeholder will be replaced by a file name -->
+    <string name="send_file_to">Send \"%1%s\" to…</string>
+    <string name="send_file_to">Send Message to…</string>
```

Comment:
dumb question: is it correct that there are two duplicate names for the two strings?

Reply:
no, this was my fault :)